packages/website/plugins/generated-rule-docs.ts

```diff
@@ -299,6 +299,33 @@ export const generatedRuleDocs: Plugin = () => {
       }
     }
 
+    // Insert default rule options for ban-types
+    if (file.stem === 'ban-types') {
```

Comment:
😬 I do dislike having such rule-specific things in this already kind of big file... what do you think about having an mdx component like `<BanTypesDefaultOptions />`?

Reply:
We already have file-specific handling for this page. This cannot be an MDX component because it has to work server-side and get statically injected instead of dynamically rendered.

Reply:
Blurgh, I keep forgetting these things. MDX components don't work server-side? Is this mentioned in docs / one of the things blocked by MDX v2? (I searched and can't find it)

Reply:
MDX components are basically React components. They can get server-rendered but they can't access server data like files.